dataland-frontend/src/components/forms/parts/elements/derived/ProcurementCategoryFormElement.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <div class="form-field">
+    <div data-test="dataPointToggle" class="form-field border-none vertical-middle">
+      <InputSwitch
+        data-test="dataPointToggleButton"
+        inputId="dataPointIsAvailableSwitch"
+        @click="this.isItActive = false"
+        v-model="isItActive"
+      />
+      <h5 data-test="dataPointToggleTitle" class="m-2">
+        {{ label }}
+      </h5>
+    </div>
+
+    <FormKit type="group" :name="name" v-if="isItActive">
+      <div>
+        <div class="form-field border-none">
+          <NaceCodeFormFieldBindData
+            label="Definition Product Type/Service"
+            description="Define the procured product types/services per category (own operations)"
+            name="procuredProductTypesAndServicesNaceCodes"
+            v-model:selectedNaceCodesBind="procuredProductTypesAndServicesNaceCodesValue"
+          ></NaceCodeFormFieldBindData>
+        </div>
+
+        <div class="form-field border-none">
+          <UploadFormHeader
+            label="Order Volume per Procurement %"
+            description="State your order volume per procurement category in the last fiscal year (percentage of total volume) (own operations)"
+            :is-required="false"
+          />
+          <FormKit
+            type="text"
+            v-model="percentageOfTotalProcurementValue"
+            name="percentageOfTotalProcurement"
+            validation="number"
```

Comment:
Two things here:
1.) We should validate that the percentage is between 0 and 100.  Other numbers are not possible from a business perspective.

2.) Currently if you type in "12", the POST request sends "12".   It should send 0.12 !
Compare this to the POST requests in the eutaxonomy-nonfinancials upload page.  If I type "12" there, it sends 0.12 .

Reply:
We also have a PercentageFormField that can be used here.

Reply:
yes, right.

Reply:
Yes great idea Florian.   And I think `PercentageFormField` also provides the desired validation.

Reply:
I think for "representedEmployees" this is broken too in Lksg.  Or at least inconsistent compared to EuTaxonomyNonFinancials for example.

Reply:
As discussed in the daily today, we won't fix the inconsistent percentage submission.
So the only thing to consider doing here is to reuse `PercentageFormField` as suggested by @fsoffel

Reply:
Since we disable and enable this block, we would need a new component that would not store the data itself, but only propagate it to the parent. If we do not change this value (from 22 to 0.22 and again from 0.22 to 22), Lets leaving it until the final solution?

Reply:
Ok I see. Then let's leave it like it is now.

Anyways I'd like to discuss something:

We could also enable our existing components - e.g. `PercentageFormField` or `MultiSelectFormField` - to send its current value to the parent and also receive it from the parent.  Then we could use those components instead of using the "Bind"-versions of them.
In cases where we don't need that send-and-receive-behaviour - e.g. in the `CreateLksg` component - we could just not use the `v-model` in the parent and therefore "cut" the connection between parent and child.

@MarcinStanczyk7  do you think this would be possible in general?

Reply:
Could you try out my idea for maybe for the `PercentageFormField`?
If it works, we could also use it to avoid needing `NaceCodeBind`.  We still have enough time in this sprint for it I think.

Reply:
PercentageFormField component works as described by Emanuel and is committed. Multiselect and NaceCode, however, have their states in data(), we cannot hold the state in computed and in data at the same time, so for now we will stay with the Bind version and without Bind. We can transfer their state to Pinia and manage them from there, but it generates more bugs for me so just leave it like it is because I don't want to commit changes before the weekend and mess up a good sprint. I only committed the PercentageFormField.

Reply:
Ok thx I will have a look now.

The name is still the old one btw:

![image](https://github.com/d-fine/Dataland/assets/99255296/64308a50-43c3-403c-8233-45fc6da126ea)

Reply:
Yes, sorry for that, corrected

Reply:
For MultiSelectFormFIeld I agree that we need the "Bind" version, because the "Bind" version does not submit anything via FormKit, but the normal version does.

For NaceCodeFormField however, I don't see why it is a problem:
Can't you do it the same way as in `PercentageFormField` ?   It does basically the same thing right?

Reply:
We should replace:
  data() selectedNaceCodes by computed selectedNaceCodes

Reply:
Yes, also see my screenshot in the other conversation I just opened.   This migth work and then we can remove NaceCodeFormFieldBindData

If it doesn't work, I'll resolve this conversation and we'll leave it as it is.

Reply:
👍  it worked so I am closing this